src/components/Camera/style.css

```diff
@@ -8,38 +8,30 @@
   justify-content: space-between;
 }
 
-.fullScreenContainer() {
+.fullScreenContainer {
   position: static;
-  .uploadFallback {
-    display: none;
-  }
 }
 
-.video-overlay {
-  /* Relatively positioned so absolute children are bound to this overlay */
+.nonFullScreenContainer {
```

Comment:
You could also just do `:not(.fullScreenContainer)` (supported by pretty much all browsers)

Reply:
I won't be using `:not` in this instance but I've changed the class name to something more meaningful.